ee/tabby-webserver/src/handler.rs

```diff
@@ -9,62 +9,82 @@ use axum::{
 use hyper::{Body, StatusCode};
 use juniper_axum::{graphiql, graphql, playground};
 use tabby_common::api::{code::CodeSearch, event::RawEventLogger, server_setting::ServerSetting};
+use tabby_db::DbConn;
```

Comment:
could you fix the lint here? it should be straight forward to modify https://github.com/TabbyML/tabby/blob/60310d416177de629853d06e746d1a058ffd9a7b/rules/only-service-can-depend-tabby-db.yml#L8

(adding a ignore line for `handler.rs`)